x-pack/plugins/security_solution/cypress/integration/host_details/risk_tab.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
```

Comment:
Offtopic: The file is missing `.spec` extension.

Reply:
Was like that before and I think it was a reason behind for it. I'll add it then since if I'm not mistaken you were the author of this part ;)